birdwatcher.py

```diff
@@ -47,6 +52,11 @@ def parse_show_protocols(f):
         yield ('bird_up{bird_protocol_instance="%s"}' % protocol_instance,
                int(line[3] == 'up'))
 
+        info = line[5] if len(line)>5 else "unknown"
+        yield bird_info_line(protocol_instance, info, "unknown")
```

Comment:
Thanks for the comment. 
How do you recommend doing that ? 

bird_info{{bird_protocol_instance="Mesh_10_101_4_114"} "Established" 
Is not a valid prometheus metric line. According to the documentation the value needs to be a float. 

[Here](https://prometheus.io/docs/instrumenting/exposition_formats/) 

> metric_name [
>   "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
> ] value [ timestamp ]
> 
"value is a float"

In terms of the generated metrics I see two options:

1) 
This is what this PR has. I can extend it with the other parameters in bird_state_names
bird_info{bird_protocol_instance="Mesh_10_101_4_114",value="unknown"}  0
bird_info{bird_protocol_instance="Mesh_10_101_4_114",value="Active"} 1
bird_info{bird_protocol_instance="Mesh_10_101_4_114",value="Established"} 0
2) I got the inspiration from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/blob/master/collectors/node.go#L256)  
bird_established{bird_protocol_instance="Mesh_10_101_4_114",condition="true"} 0
bird_established{bird_protocol_instance="Mesh_10_101_4_114",condition="false"} 1
bird_established{bird_protocol_instance="Mesh_10_101_4_114",condition="unknown"} 0
bird_active{bird_protocol_instance="Mesh_10_101_4_114",condition="true"} 1
bird_active{bird_protocol_instance="Mesh_10_101_4_114",condition="false"} 0
bird_active{bird_protocol_instance="Mesh_10_101_4_114",condition="unknown"} 0

In either case, I think the values in the bgp_state_names should be known in the birdwatcher.py implementation.

The birdwatcher script already knows about other parameters from bird. Is adding this a big problem ? 

Do you have any other option ? 

PS: I do not think emitting only the current value of the bird info ( the true value) is a viable option. 
For example if the only emitted metric is 
bird_info{bird_protocol_instance="Mesh_10_101_4_114",value="Active"} 1
How would the consumer of these metrics can trigger on a protocol not being Established? I am not sure whether there is a way to distinguish between a false value and a non-existing value.